detect_secrets/plugins/base.py

```diff
@@ -239,7 +239,7 @@ def adhoc_scan(self, string):
 
         return output[verified_result]
 
-    def verify(self, token, content=''):
```

Comment:
It's actually meant to be `context`.

A couple of lines down, it reads:

```
:param context: lines of context around identified secret
```

Reply:
Right, that's why I renamed it to `context`.

Reply:
🤦 I must be getting cross-eyed.